packages/headless/package.json

```diff
@@ -23,14 +23,24 @@
   "license": "ISC",
   "dependencies": {
     "@nestjs/common": "^8.4.0",
+    "@nestjs/config": "^2.0.0",
     "@nestjs/core": "^8.4.0",
+    "@nestjs/jwt": "^8.0.0",
+    "@nestjs/passport": "^8.2.1",
     "@nestjs/platform-express": "^8.0.0",
+    "@nestjs/swagger": "^5.2.1",
+    "@types/bcrypt": "^5.0.0",
+    "bcrypt": "^5.0.1",
     "joi": "^17.6.0",
     "mongoose": "^6.2.2",
+    "passport": "^0.5.2",
+    "passport-jwt": "^4.0.0",
+    "passport-local": "^1.0.0",
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.2",
     "rxjs": "^7.2.0",
-    "sequelize-typescript": "^2.1.3"
+    "sequelize-typescript": "^2.1.3",
+    "swagger-ui-express": "^4.3.0"
```

Comment:
do we need this although nestjs/swagger is used

Reply:
swagger-ui-express is required for swagger